assign2/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+	margin: 0;
+	padding: 0;
+	font-family: Raleway;
+}
+
+.container {
+	font-weight: 300;
+	display: flex;
+	justify-content: space-around;
+}
+
+.container p {
+	color: grey;
+}
+
+h1 {
+	margin-top: 50px;
+	margin-left: 170px;
+	margin-bottom: 50px;
+}
+
+.col-1 {
+	width: 60%;
+	margin-left: 50px;
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-between;
```

Comment:
why did we use this property? What happens if we don't use this?

Reply:
sir row wrap to make all div align with row and if any div does not fit in that row it should automatically move to next row and justify-content to provide space between the div.